shim/shim.c

```diff
@@ -432,6 +446,7 @@ handle_proxy_output(struct cc_shim *shim)
 		code = *(buf + STREAM_HEADER_SIZE); 	// hyperstart has sent the exit status
 		shim_debug("Exit status for container: %d\n", code);
 		free(buf);
+		restoreTerminal();
```

Comment:
@dlespiau: Sure :) Looks like I had been staring at go code long enough :)
@jodh-intel I'll look at adding it at atexit